test-utils import using incorrect relative path

**Intended outcome:**

Successfully use `MockedProvider` from `test-utils.js` in a test.

**Actual outcome:**

```
● Test suite failed to run
    Cannot find module '../src' from 'test-utils.js'
      at Resolver.resolveModule (../node_modules/jest-resolve/build/index.js:179:17)      at Object.<anonymous> (../node_modules/react-apollo/test-utils.js:19:13)
```

**How to reproduce the issue:**

Attempt to import from `test-utils.js` for testing.

**Version**

* apollo-client@^2.2.5
* react-apollo@^2.1.0-beta-2

`git bisect` investigation shows that [this change](https://github.com/apollographql/react-apollo/blame/master/src/test-utils.tsx#L5) to the `ApolloProvider` import in `2b26e8b` is not correctly compiling in UMD. Compiling on any commit after that change keeps `../src` as the import. Changing import to `./` appears to fix the issue.

--- a/src/test-utils.tsx
+++ b/src/test-utils.tsx
@@ -2,7 +2,7 @@ import * as React from 'react';
 import ApolloClient from 'apollo-client';
 import { InMemoryCache as Cache } from 'apollo-cache-inmemory';
 
-import { ApolloProvider } from '../src';
+import { ApolloProvider } from './';
 import { mockSingleLink } from './test-links';
 export * from './test-links';
 

